lib/index.js

```diff
@@ -65,21 +66,29 @@ exports.lint = function(opts) {
 };
 
 function allSources(basedir, pkg, additional) {
-  var sources = [];
-  if (pkg.main) {
-    sources.push(pkg.main);
-  }
-  if (pkg.bin) {
-    sources = sources.concat(_.values(pkg.bin));
-  }
-  if (additional) {
-    sources = sources.concat(additional);
-  }
-  return sources.map(function(file){
+  return listify(pkg.main)
+  .concat(_.values(pkg.bin? pkg.bin : [] ))
```

Comment:
Could this be `.concat(_.value(pkg.bin))` ? It seems it returns `null`, `undefined` and `{}` all return an empty array.